core-web/package.json

```diff
@@ -158,8 +158,10 @@
         "@nx/react": "18.0.4",
         "@nx/rollup": "18.0.4",
         "@nx/storybook": "18.0.4",
+        "@nx/vite": "18.0.4",
+        "@nx/web": "18.0.4",
```

Comment:
Why web?

Reply:
`@nx/web` is used by Nx to generate libraries, which I think was added when I added the experiment lib.